src/components/Protected/Permissions.js

```diff
@@ -0,0 +1,108 @@
+//check if user is logged in and has a token
+
+    export function isUser()  {  
+
+        const user = JSON.parse(localStorage.getItem("user"));
```

Comment:
our computers might not have this local storage yet;
so we need to handle when blank

What if you fall back to an object when blank?

this is just an example:
`const user = JSON.parse(localStorage.getItem("user")) || {accessToken:  ""}`

please change what's in the fallback object to what you actually might need

Reply:
thanks for bringing this to my attention. The problem is because localstorage isnt created until you login so it sends a value of null. Handling  that error properly should fix it.